test/integration/crl_test.go

```diff
@@ -1,3 +1,87 @@
 //go:build integration
 
 package integration
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/letsencrypt/boulder/core"
+	"github.com/letsencrypt/boulder/test"
+)
+
+// runUpdater executes the crl-updater binary with the -runOnce flag, and
+// returns when it completes.
+func runUpdater(t *testing.T) {
+	t.Helper()
+
+	config_dir := os.Getenv("BOULDER_CONFIG_DIR")
+	if config_dir == "" {
+		t.Fatal("couldn't determine configuration directory")
+	}
```

Comment:
You could check that the environment variable here, by checking the returned `bool` var. `os.Envget()` returns `syscall.Envget()` which returns `"", false` for empty values.

Since you've got `configDir` here, you can pass it to each instance call of `runUpdater()`. If you would still rather fetch the config directory here and in `runUpdater()` I would recommend making `"BOULDER_CONFIG_DIR"` a constant.

```suggestion
func runUpdater(t *testing.T) {
	t.Helper()

	configDir, ok := os.Getenv("BOULDER_CONFIG_DIR"))
	test.Assert(t, ok, "'BOULDER_CONFIG_DIR' not set")

    // The crl-updater and crl-storer are not yet deployed in Prod, so only run
	// this test in config-next.
	if !strings.Contains(configDir, "config-next") {
		return
	}
```